azurerm/internal/services/sql/tests/resource_arm_sql_server_test.go

```diff
@@ -366,3 +396,85 @@ resource "azurerm_sql_server" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
+
+func testAccAzureRMSqlServer_withBlobAuditingPolices(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%[1]d"
+  location = "%[2]s"
+}
+
+resource "azurerm_storage_account" "test" {
+  name                     = "str%[1]d"
```

Comment:
we always want to start name with acctest
```suggestion
  name                     = "acctest%[2]d"
```
in other resources you can see use of data.IntOfLen()